vtex-apps/store-components
# Add labelListPrice props in ProductPrice

#### What is the purpose of this pull request?
Add `labelListPrice` to the `ProductPrice` component, following these rules:

- If it's an empty string, no label will be showed.
- If it's undefined or null, the default label will be showed.
- Otherwise, show the label passed.

#### What problem is this solving?
Discount price labels should be flexible and changed in the `blocks` file.

#### How should this be manually tested?
[Here](https://rubacao--storecomponents.myvtex.com/traveler-backpack/p?disableUserLand)

If you want to see other behaviors, edit the blocks in `store-theme` and use the flag `?disableUserLand` in the url.

#### Types of changes
